src/player.cpp

```diff
@@ -5748,6 +5748,133 @@ static int bound_mod_to_vals(int val, int mod, int max, int min)
     return mod;
 }
 
+void player::print_health()
+{
+    if( !is_player() ) {
+        return;
+    }
+    int current_health = get_healthy();
+    if( has_trait( "SELFAWARE" ) ) {
+        add_msg( "Your current health value is: %d", current_health );
+    }
+    int roll = rng(0,4);
+    std::string message = "";
```

Comment:
And for the record (this appears several times in the code already), a `std::string` is automatically initialized to an empty string, there is no need to do this manually via ` = ""`.